spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1036,6 +1037,8 @@ def _run_algorithms(
             self._txrx = executor.get_item("MemoryTransceiver")
             self._machine_allocation_controller = executor.get_item(
                 "MachineAllocationController")
+            report_folder = executor.get_item("ReportFolder")
```

Comment:
Why not just use
self._report_default_directory

That is setup in the __init__

Reply:
Feel free to change the code!